build.go

```diff
@@ -150,8 +152,14 @@ func Build(build BuildFunc, options ...Option) {
 	}
 	config.logger.Debugf("Buildpack: %+v", ctx.Buildpack)
 
-	API := strings.TrimSpace(ctx.Buildpack.API)
-	if API != "0.5" && API != "0.6" && API != "0.7" {
+	apiV05, _ := semver.NewVersion("0.5")
+	apiV06, _ := semver.NewVersion("0.6")
+	apiVersion, err := semver.NewVersion(ctx.Buildpack.API)
+	if err != nil {
+		config.exitHandler.Error(err)
+		return
+	}
+	if apiVersion.LessThan(apiV05) {
```

Comment:
I think this is something that didn't get ported over from v1.

https://github.com/buildpacks/libcnb/pull/134

That PR needs to be applied to v2 as well, before we can merge this. Otherwise, I think it's going to create merge conflicts.

Reply:
Just added this construct but none of the other 0.8 API changes. Let me know if this will suffice.

Reply:
Give me a min, I'll port #134 over. I think if you only add this bit, it'll make merging 134 more difficult. If we merge that over first, then rebase your PR, I think these diffs should just disappear.

Reply:
Ok, I have #142 that ports the recent changes from main into version-2. My suggestion would be that we merge that first and rebase this PR off that. It does have a few changes to build, but I think there will be fewer conflicts if we go in this order.

Reply:
Sounds good to me.